CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.0] 2018-09-26
```

Comment:
I believe this should be 0.2.0 according to the semver guidlines.

> MINOR version when you add functionality in a backwards-compatible manner

Reply:
@sebastianfairchild  I had the same reasoning as Kenny

Reply:
if anything this seems like a patch, the previous hex value was incorrect.  and the api is still the same since the token hasn't changed. adding all of the new tokens is reason for bumping the minor version.